client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -356,6 +356,27 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "enableDartNullSafety": {
+        "description": "Generate Dart models with null safety for Flutter applications using DataStore. Refer to https://docs.amplify.aws/lib/project-setup/null-safety/q/platform/flutter for more information",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "5.1.0",
+        "deprecationDate": "Jan 1st 2022",
+        "values": [
+          {
+            "value": "true",
+            "description": "[Recommended] Generate Dart models with null safety. Minimum version for Amplify library: amplify-flutter@0.2.0",
```

Comment:
`amplify-flutter@0.2.0` might be confusing for Dart/Flutter folks. Maybe `^0.2.0` or just `0.2.0`.

Reply:
+1 for `0.2.0` since the wording already contains "Minimum version"

Reply:
Agreed